client/api/combined.ts

```diff
@@ -24,7 +24,9 @@ function convertCombinedParams(params: ISearchParams): Partial<ISearchAPIParams>
         calendars: cvsToString(params.calendars),
         agendas: arrayToString(params.agendas),
         include_associated_planning: params.include_associated_planning,
-        source: cvsToString(params.source, 'id')
+        source: cvsToString(params.source, 'id'),
+        coverage_user_id: params.coverage_user_id,
+        coverage_user_name: params.coverage_user_name,
```

Comment:
There is no need to pass `coverage_user_name` as a search param, as it doesn't do anything.

Instead you would need to pass in the current `coverage_user_id` param into the `PlanningListSubNavComponent` using the `mapStateToProps`. Then in this component, you would get the name of the User based on the above prop and the `users` prop.

See how this is currently done with other params in the same component:
* https://github.com/superdesk/superdesk-planning/blob/develop/client/apps/Planning/PlanningListSubNav.tsx#L39
* https://github.com/superdesk/superdesk-planning/blob/develop/client/selectors/main.ts#L89